### DISCUSSION TITLE
Setup to download a single models profile?

### DISCUSSION THREAD

**Discussion by Shingotink** [UltimaHoarder/UltimaScraper#1161] [Q&A]
I am having an issue where I get the welcome line then the program hangs for over 20 minutes (I assume because I am subscribed to 1000+ profiles) it never took this long before the site changed.

If I set scrape_names to false obviously it rushes back to the original choose profile command.

Ideally I would like to get this loading time way down, but if that can't be done is there a way that I can setup to scrape a specific model? I would rather change a few lines than have to wait 20+ minutes between each scrape.

**Comment by LunarPenguin** [comment]
This seems to be a duplicate of your other discussion at #1168 please refer to my answer in that discussion
